docs/source/technical_tutorials/working_with_r/access_aws_maap.ipynb

```diff
@@ -9,7 +9,7 @@
     "\n",
```

Comment:
<p>This warning did not appear before:</p>

<pre class="ql-syntax" spellcheck="false">Warning message: 
“no simple feature geometries present: returning a data.frame or tbl_df” 
</pre><p>I'm wondering if this was caused by surrounding <code>=</code> with  spaces here:</p>

<pre class="ql-syntax" spellcheck="false">X_POSSIBLE_NAMES = lon", "Y_POSSIBLE_NAMES = lat"
</pre><p>You likely should <em>not</em> be surrounding the <code>=</code> signs with spaces within those strings.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/476/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>